_includes/v3.8/release-notes/v3.8.2-release-notes.md

```diff
@@ -0,0 +1,5 @@
+14 Aug 2019
+
+#### Bug fixes
+
+ - Fix race condition where a non-Calico iptables rule may be removed (@lmm) [felix #2102](https://github.com/projectcalico/felix/pull/2102)
```

Comment:
Previous releases have the attribution last (move the `(@lmm)` to the end of the line. (Same for the other PRs for this bug fix.)

Reply:
Thanks, fixed!